handlers/Contact/components/Locations/styles.css

```diff
@@ -12,6 +12,17 @@
   padding: 0;
 }
 
+.ContactMap{
```

Comment:
This fails the Suit validator.

* https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname
* https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname-descendentname